lib/spack/llnl/util/filesystem.py

```diff
@@ -251,6 +251,11 @@ def path_contains_subdirectory(path, root):
 @memoized
 def file_command(*args):
     """Creates entry point to `file` system command with provided arguments"""
+    if sys.platform == "win32":
+        import spack.bootstrap
+
+        with spack.bootstrap.ensure_bootstrap_configuration():
+            spack.bootstrap.ensure_resource("file")
```

Comment:
Please, let's not add more `llnl->spack->llnl` circular deps

Reply:
Moved everything in `llnl.util.filesystem` that requires an import from core spack to a core spack module.